Frameworks/CoreGraphics/D2DWrapper.mm

```diff
@@ -32,4 +32,18 @@ HRESULT _CGGetWICFactory(IWICImagingFactory** factory) {
     static HRESULT sHr = CoCreateInstance(CLSID_WICImagingFactory, nullptr, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(&sWicFactory));
     sWicFactory.CopyTo(factory);
     RETURN_HR(sHr);
+}
+
+HRESULT _CGConvertD2DGeometryToFillMode(ID2D1Geometry* geometry, D2D1_FILL_MODE fillMode, ID2D1Geometry** pNewGeometry) {
```

Comment:
Indeed. We absolutely should; this should be part of CGPath, but CGPath's default is backwards from CG's default right now.

I'll file an issue to track it and its removal here.

---
In reply to: [87899206](https://github.com/Microsoft/WinObjC/pull/1368#discussion_r87899206) [](ancestors = 87899206)